BUILD

```diff
@@ -0,0 +1,35 @@
+load("@rules_java//java:defs.bzl", "java_binary", "java_lite_proto_library", "java_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+proto_library(
+  name = "incomplete_example_proto",
+  srcs = ["java/com/google/location/lbs/activity/audioset/dataprocessing/incomplete-example.proto"],
+  deps = ["@com_google_protobuf//:timestamp_proto"],
+)
+
+java_proto_library(
+  name = "incomplete_example_java_proto",
+  deps = [":incomplete_example_proto"],
+)
+
+java_binary(
```

Comment:
Only the final executable should be a "java_binary"

Reply:
Ah. Thanks!